.github/workflows/build_and_test_dbt.yaml

```diff
@@ -6,6 +6,16 @@ on:
   push:
     branches: [master]
 
+  workflow_dispatch:
+    inputs:
+      models:
+        required: true
+        description: >
+          A space-separated list of dbt models to build.
+          Prefix with the + sign to rebuild upstream models
+        default: 'default.vw_pin_sale default.vw_pin_universe'
```

Comment:
**[Question, non-blocking]** Any particular reason to default to these two models? And will this default ever be executed, or is it a no-op given that the variable is required?

Reply:
Hard to test since I can't seen the dispatch button on non-master branches, but I think this just auto-populates the field with those views, while the `required` arg just won't let you submit the dispatch with an empty string.

Reply:
Ah, OK! I think it makes sense to store an example value here as a prompt then.